libr/core/cmd_seek.c

```diff
@@ -138,372 +138,405 @@ R_API int r_core_lines_initcache (RCore *core, ut64 start_addr, ut64 end_addr) {
-			"s", "", "Print current address",
-			"s:", "pad", "Print current address with N padded zeros (defaults to 8)",
-			"s", " addr", "Seek to address",
-			"s-", "", "Undo seek",
-			"s-", " n", "Seek n bytes backward",
-			"s--", "", "Seek blocksize bytes backward",
-			"s+", "", "Redo seek",
-			"s+", " n", "Seek n bytes forward",
-			"s++", "", "Seek blocksize bytes forward",
-			"s[j*=]", "", "List undo seek history (JSON, =list, *r2)",
-			"s/", " DATA", "Search for next occurrence of 'DATA'",
-			"s/x", " 9091", "Search for next occurrence of \\x90\\x91",
-			"s.", "hexoff", "Seek honoring a base from core->offset",
-			"sa", " [[+-]a] [asz]", "Seek asz (or bsize) aligned to addr",
-			"sb", "", "Seek aligned to bb start",
-			"sC", "[?] string", "Seek to comment matching given string",
-			"sf", "", "Seek to next function (f->addr+f->size)",
-			"sf", " function", "Seek to address of specified function",
-			"sg/sG", "", "Seek begin (sg) or end (sG) of section or file",
-			"sl", "[?] [+-]line", "Seek to line",
-			"sn/sp", "", "Seek next/prev scr.nkey",
-			"so", " [N]", "Seek to N next opcode(s)",
-			"sr", " pc", "Seek to register",
-			//"sp [page]  seek page N (page = block)",
+			"Usage: ss", "", " # Seek silently commands (without adding the address to the seek history)",
+			"ss", " addr", "Seek silently to address",
+			"ssr", " pc", "Seek silently to register",
 			NULL
 			};
-			r_core_cmd_help(core, help_message);
-		}
+			r_core_cmd_help (core, help_message);
+			}
+			break;
+		default:
+			r_core_seek (core, off * sign, 1);
+			r_core_block_read (core);
 			break;
 		}
-	} else r_cons_printf ("0x%"PFMT64x"\n", core->offset);
+
+		break;
+	case '?': {
+		const char * help_message[] = {
+		"Usage: s", "", " # Seek commands",
+		"s", "", "Print current address",
+		"s:", "pad", "Print current address with N padded zeros (defaults to 8)",
+		"s", " addr", "Seek to address",
+		"s-", "", "Undo seek",
+		"s-", " n", "Seek n bytes backward",
+		"s--", "", "Seek blocksize bytes backward",
+		"s+", "", "Redo seek",
+		"s+", " n", "Seek n bytes forward",
+		"s++", "", "Seek blocksize bytes forward",
+		"s[j*=]", "", "List undo seek history (JSON, =list, *r2)",
+		"s/", " DATA", "Search for next occurrence of 'DATA'",
+		"s/x", " 9091", "Search for next occurrence of \\x90\\x91",
+		"s.", "hexoff", "Seek honoring a base from core->offset",
+		"sa", " [[+-]a] [asz]", "Seek asz (or bsize) aligned to addr",
+		"sb", "", "Seek aligned to bb start",
+		"sC", "[?] string", "Seek to comment matching given string",
+		"sf", "", "Seek to next function (f->addr+f->size)",
+		"sf", " function", "Seek to address of specified function",
+		"sg/sG", "", "Seek begin (sg) or end (sG) of section or file",
+		"sl", "[?] [+-]line", "Seek to line",
+		"sn/sp", "", "Seek next/prev scr.nkey",
+		"so", " [N]", "Seek to N next opcode(s)",
+		"sr", " pc", "Seek to register",
+		"ss", "", "Seek silently (without adding an entry to the seek history)",
```

Comment:
`ssr` is explained in `ss?` as usual. https://github.com/radare/radare2/pull/7052/files#diff-64a3e927280b6b6489f8bca50c35f2e6R494